ui/app/src/components/shared/ConfirmCopyUrlToClipboard.test.tsx

```diff
@@ -0,0 +1,246 @@
+import React from "react";
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import { ConfirmCopyUrlToClipboard } from "./ConfirmCopyUrlToClipboard";
+import { Resource } from "../../models/resource";
+import { ResourceType } from "../../models/resourceType";
+
+// Mock FluentUI components
+vi.mock("@fluentui/react", async () => {
```

Comment:
A lot of Fluent UI mocking in each individual test, can this be consolidated and re-used?

Reply:
Had a  go at sorting that. What you think?